bff-node/src/config/passport.ts

```diff
@@ -0,0 +1,158 @@
+import { Issuer, Strategy as OpenIDStrategy, TokenSet } from 'openid-client';
+import './env';
+import { IDPortenProfile } from '../types/types';
+import { Profile } from '../entities/Profile';
+import { ProfileRepository, SessionRepository } from '..';
+import passport from 'passport';
+import { readCookie } from '../util/sessionUtils';
+import { SessionData } from '../entities/SessionData';
+
+export const initPassport = async () => {
+  if (!SessionRepository) throw new Error('SessionRepository not initialized');
+  if (!ProfileRepository) throw new Error('ProfileRepository not initialized');
+  if (!process.env.ISSUER_URL) {
```

Comment:
I would try to move all these into a `config.ts` where we could parse and validate all these environment variables in one place, and remove the use of `process.env` in all other places. E.g:

```typescript

function assertEnvVariable(envVariable: string, defaultValue?: string) {
  if (
    process.env[envVariable] === null ||
    process.env[envVariable] === undefined
  ) {
    if (defaultValue !== undefined) {
      console.info(
        `No value found for ${envVariable}. Using default value: ${defaultValue}.`
      )
      return defaultValue
    } else {
      console.error(
        `No value or default value found for environment variable ${envVariable}. Exiting.`
      )
      process.exit(1)
    }
  }
  return process.env[envVariable] as string
}

export default {
  name: assertEnvVariable('APP_NAME'),
  environment: assertEnvVariable('NODE_ENV', 'development'),
  auth: {
    issuer: assertEnvVariable('ISSUER'),
    clientId: assertEnvVariable('CLIENT_ID') 
    ...
  },
}

```